samples/develop/HuaweiCloud-Compute-opentofu-dev.yml

```diff
@@ -422,11 +422,6 @@ deployment:
       override_special = "#%@"
     }
 
-    resource "huaweicloud_kps_keypair" "keypair" {
```

Comment:
The `admin_pass` configuration will not take effect when using keypair? Let me test on the FlexibleEngine

Reply:
@baixinsui yes correct.